packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts

```diff
@@ -0,0 +1,174 @@
+    adminACLContract: "0x0000000000000000000000000000000000000000",
+    // optionally define this to set default vertical name for the contract after deployment.
+    // if not defined, the default vertical name will be "unassigned".
+    // common values include `studio`, `fullyonchain`, `flex`, or partnerships like `artblocksxpace`.
+    // also note that if you desire to create a new vertical, you will need to add the vertical name to the
+    // `project_verticals` table in the database before running this deploy script.
+    defaultVerticalName: "studio",
+  },
+  {
+    // 0 for Engine, 1 for Engine Flex
+    engineCoreContractType: 1,
+    // prod-only: see efficient_addresses...txt in network directories for a list of efficient salts
+    salt: "0x0000000000000000000000000000000000000000a73bf3db9deb3358e3080020",
+    // INCREMENT THESE NUMBERS FOR EACH NEW STUDIO DEPLOYMENT
+    tokenName: "Art Blocks Studio | 29",
+    tokenTicker: "ABSTUDIO_29",
+    // set to the address you want to transfer the superAdmin role to
+    // (can be the zero address if you have an existing Admin ACL Contract and are not transferring)
+    newSuperAdminAddress: "0x584Dc29ddd6cA2a3200dD630dBF65B6A84Cb0d86",
+    // see /scripts/util/constants.ts::MAIN_CONFIG for the correct address if prod deployment
+    renderProviderAddress: "0x036F3D03C1ccdde1878F01607922EA12110Ee9Bd",
+    // platform provider address should be set to the zero address for Studio deployments
+    platformProviderAddress: "0x0000000000000000000000000000000000000000",
+    startingProjectId: 0,
+    // autoApproveArtistSplitProposals should be true for Studio deployments
+    autoApproveArtistSplitProposals: true,
+    // nullPlatformProvider must be true for Studio deployments
+    nullPlatformProvider: true,
+    // allowArtistProjectActivation must be true for Studio deployments
+    allowArtistProjectActivation: true,
+    // if you want to use an existing admin ACL, set the address here (otherwise set as the zero address to deploy a new one)
+    adminACLContract: "0x0000000000000000000000000000000000000000",
+    // optionally define this to set default vertical name for the contract after deployment.
+    // if not defined, the default vertical name will be "unassigned".
+    // common values include `studio`, `fullyonchain`, `flex`, or partnerships like `artblocksxpace`.
+    // also note that if you desire to create a new vertical, you will need to add the vertical name to the
+    // `project_verticals` table in the database before running this deploy script.
+    defaultVerticalName: "studio",
+  },
+  {
+    // 0 for Engine, 1 for Engine Flex
+    engineCoreContractType: 1,
+    // prod-only: see efficient_addresses...txt in network directories for a list of efficient salts
+    salt: "0x0000000000000000000000000000000000000000a73bf3db9deb5f1cfa0a0080",
```

Comment:
this needs to be removed from the flex efficient addresses file 😃

Reply:
thank you !!!